pkgs/top-level/python-packages.nix

```diff
@@ -4453,7 +4453,7 @@ in {
 
     propagatedBuildInputs = with self; [
       six keystoneclient prettytable oslo-utils oslo-i18n lxml httplib2 cliff
-      dogpile_cache appdirs anyjson pbr openstackclient
+      dogpile_cache appdirs anyjson pbr pkgs.openstackclient
```

Comment:
`openstackclient` brings python modules that are required by `ironicclient`. So, it had to be in `propagatedBuildInputs`. But `ironicclient` is currently broken because `cliff` is. I will simply mark ironicclient as broken and explain why in a commen.

In fact, `ironicclient` should be removed and added to the `openstaclient` one... but I personally really don't need it.